### ISSUE TITLE
copy code block buttons are not accessible to screen readers

### ISSUE THREAD

**Issue by dmuelle** [OpenLiberty/openliberty.io#2885]
While doing JAWS testing on the Open Liberty docs, I found that the "copy code block" buttons are not discoverable by the screen reader. 

Using the "B" key in the JAWS program takes you through all the buttons on the current screen. On multiple pages, I found that this command never finds the code copy buttons. Also, If you hover the over the buttons, the screen reader reads the source file designation for the code example (eg "XML") rather than the alt text for the copy code button. 

It appears there is no way that someone using  a screen reader would know the copy code buttons exist.

tested multiple pages/ code blocks on chrome on a mac
